### PR TITLE
create filters

### LINKED ISSUE #5: create filters

### PR DESCRIPTION
**Pull request by JoannaTabbi**
Close #5

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
           <input type="submit" value="Rechercher" />
         </form>
       </section>
+      <section id="filters">
+        <h4>Filtres</h4>
```
Comment (Tutt-J) [inline index.html]: Why h4 ?

File: index.html [hunk 2]
```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
...
+        <ul>
+          <li><i class="fas fa-money-bill-wave"></i>Economique</li>
```
Comment (Tutt-J) [inline index.html]: Add link on word

File: index.html [hunk 3]
```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
...
+          <li><i class="fas fa-child"></i>Familial</li>
```
Comment (Tutt-J) [inline index.html]: Add link on word

File: index.html [hunk 4]
```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
...
+          <li><i class="fas fa-heart"></i>Romantique</li>
```
Comment (Tutt-J) [inline index.html]: Add link on word

File: index.html [hunk 5]
```diff
@@ -36,6 +36,16 @@ <h1>Trouvez votre hébergement pour des vacances de rêve</h1>
...
+          <li><i class="fas fa-dog"></i>Animaux autorisés</li>
```
Comment (Tutt-J) [inline index.html]: Add link on word

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: to differentiate between the filters' title and the text below ("Plus de 500...")

File: index.html (hunk 2 above)
Comment (JoannaTabbi) [reply index.html]: Is it only on the word or the whole line (word + icon) ?

File: index.html (hunk 3 above)
Comment (JoannaTabbi) [reply index.html]: Is it only on the word or the whole line (word + icon) ?

File: index.html (hunk 4 above)
Comment (JoannaTabbi) [reply index.html]: Is it only on the word or the whole line (word + icon) ?

File: index.html (hunk 5 above)
Comment (JoannaTabbi) [reply index.html]: Is it only on the word or the whole line (word + icon) ?

File: index.html (hunk 1 above)
Comment (Tutt-J) [reply index.html]: Yes but you haven't any h2 or h3 yet

File: index.html (hunk 2 above)
Comment (Tutt-J) [reply index.html]: Juste to word

File: index.html (hunk 3 above)
Comment (Tutt-J) [reply index.html]: Juste to word

File: index.html (hunk 4 above)
Comment (Tutt-J) [reply index.html]: Juste to word

File: index.html (hunk 5 above)
Comment (Tutt-J) [reply index.html]: Juste to word

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: actually I hesitated between h3 and h4, filters title has the same font-size as the footer's headings that we put on h4, though I'm not sure it'a a good criterion...
I was thinking of using h2 for other main sections titles ("Hébergement...", "Activités..." etc)

File: index.html (hunk 2 above)
Comment (JoannaTabbi) [reply index.html]: ok

File: index.html (hunk 3 above)
Comment (JoannaTabbi) [reply index.html]: ok, done

File: index.html (hunk 4 above)
Comment (JoannaTabbi) [reply index.html]: done

File: index.html (hunk 5 above)
Comment (JoannaTabbi) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (JoannaTabbi) [reply index.html]: ok, changed to h2, new commit arrives

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**